mne/preprocessing/ica.py

```diff
@@ -977,7 +987,7 @@ def score_sources(self, inst, target=None, score_func='pearsonr',
                                  'number of time slices.')
             # auto target selection
             if isinstance(inst, BaseRaw):
-                sources, target = _band_pass_filter(self, sources, target,
+                sources, target = _band_pass_filter(inst, sources, target,
```

Comment:
Please add a comment that we pass inst, not self because sfreq can be different, it was not obvious for me at the first sight so a comment would be useful.

Reply:
ok